### ISSUE TITLE
WCAG2Mobile: Draft feedback - Use of key term View is problematic

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#638]
The current WCAG2Mobile draft contains the following:

> Key Terms
> The glossary terms “document” and “software” in WCAG2ICT are replaced with the defined terms “screen” and “view”. 

This issue has been created for the WCAG2ICT task force to agree a position on whether the use of View in word substitutions is problematic, and if so, to agree a position to feedback to the Mobile task force.

**Comment by pday1** [comment]
Here is my proposal to start the discussion off. 

Replacing the WCAG2ICT term "software" with "view" in WCAG2Mobile is problematic for the following reasons:

- "View" is a WCAG3 term, and therefore should not be used in a note giving guidance on how to apply WCAG 2.2 to Mobile Applications. 
- "View" is a developing term in WCAG3, and therefore the meaning may change or evolve over time as the WCAG3 standard is developed.
- Use of "view" instead of "software" does not harmonise with language used in WCAG2ICT, thus causing potential confusion. 
- Use of "view" instead of "software" means that WCAG2Mobile guidance does not harmonise with standards such as EN 301 549 which have adopted WCAG2ICT language, thus causing further confusion. 
-

**Comment by ChrisLoiselle** [comment]
@pday1 great work. I agree that current state of WCAG 2.x  vs. future state (WCAG 3) is at the core of this issue around terminology WCAG2Mobile is currently referencing.

**Comment by maryjom** [comment]
Thanks @pday1. Maybe another point is that current WCAG 2.2 language also doesn't solve the problem surrounding single-page web applications, so the fact that a single software application (the current unit of conformance equated to a web page) can support view-based titles, not all software - potentially not all mobile software platforms as well, has such a view-based title attribute.

**Comment by GreggVan** [comment]
In addition-- WCAG 3 has not functional definition for VIEW.    We all love the term (and have since WCAG2ICT version 1)  but no one has ever been able to create a definition for it that others have not been able to point out LARGE holes in immediately.       No use of VIEW should be allowed in any normative work until someone successfully can define it in a way that is not so ambiguous that people cannot agree on what is or is not a view based on that definition.

**Comment by maryjom** [comment]
The AG WG is working on a definition of "view" that is in the current editor's draft
>View
>Testing scope that includes all [content](https://www.w3.org/TR/wcag-3.0/#dfn-content) visually and [programmatically](https://www.w3.org/TR/wcag-3.0/#dfn-programmatically-determinable) available without a significant change. Conceptually, views correspond to the definition of a web page as used in WCAG 2, but are not restricted to content meeting that definition. For example, a view could be considered a “screen” in a mobile app or a layer of web content, such as a modal dialog.

**Comment by GreggVan** [comment]
Hi Mike,
Just this last week the WCAG3 group came up with a definition that just MIGHT work.    Still needs some noodling but take a look at it. I think Chris L send you the new definition.  I can't find the link to it right now.   I'm not sure it will work but it is the first definition that I can't see holes in right away.     Give it a look

**Comment by ChrisLoiselle** [comment]
Chris to take action item on addressing with facilitators call to discuss next steps

**Comment by bruce-usab** [comment]
> ...Just this last week the WCAG3 group came up with a definition that just MIGHT work. 

I am pretty sure this is this WCAG3 deploy preview with [definition for view](https://deploy-preview-327--wcag3.netlify.app/guidelines/#dfn-view) is what @GreggVan  was referring to.  I found the link in the [minutes from 7/29 call](https://www.w3.org/2025/07/29-ag-minutes.html#item04).

> content that is actively available in a viewport including that which can be scrolled or panned to, and any additional content that is included by expansion while leaving the rest of the content in the viewport actively available
